Mac-App/Backend/Backend/Parsers/SwiftDepsParser/SwiftDepsParser.swift

```diff
@@ -18,9 +18,9 @@ func parseSwiftDeps(_ swiftDeps: [SwiftDeps]) -> [DependencyTree] {
                 let deps = dep.dependsTopLevel
                     .filter { $0.tag.description != "!private" }
                     .compactMap { (try? $0.represented().string) }
-                    .filter { $0 != name }
-                    .filter { systemSymbols.contains($0) == false }
-                    .filter { $0.contains("NS") == false }
+                    .filterOcurrancesOf(name)
+                    .excludeSystemSymbols()
+                    .excludeSystemSymbolsPrefixes()
```

Comment:
Seeing this again & after your good observation from yesterday regarding array iteration performance, maybe it makes sense to exclude all the symbols before calling `filterOcurrancesOf(name)` as by that time the collection is greatly reduced in size 🤔

Reply:
Yeah I think that would be better, although I'm also not sure how much performance affect would it have, maybe it is very minor, but still it would not hurt to exclude first I think @Thurman1776-

Reply:
https://github.com/Thurman1776-/Uncomplect-us/pull/2/commits/7a40f7602f03c60fdf266d6b911b556131edc94f